x/auth/types/params.go

```diff
@@ -6,32 +6,35 @@ import (
 
 // Default parameter values
 const (
-	DefaultMaxMemoCharacters      uint64 = 256
-	DefaultTxSigLimit             uint64 = 7
-	DefaultTxSizeCostPerByte      uint64 = 10
-	DefaultSigVerifyCostED25519   uint64 = 590
-	DefaultSigVerifyCostSecp256k1 uint64 = 1000
+	DefaultMaxMemoCharacters         uint64 = 256
+	DefaultTxSigLimit                uint64 = 7
+	DefaultTxSizeCostPerByte         uint64 = 10
+	DefaultSigVerifyCostED25519      uint64 = 590
+	DefaultSigVerifyCostSecp256k1    uint64 = 1000
+	DefaultSigVerifyCostEthSecp256k1 uint64 = 21000
```

Comment:
how come its more expensive than normal secp256k1

Reply:
Can make smaller, was copying ethereum.

Reply:
makes sense, can you leave a comment this is to match eth?

Reply:
Sorry this is incorrect. 


"The gas cost (for the buyer) of executing the ECDSA signature verification is 29,293 gas. This includes the 21,000 to initiate the transaction, so the ECDSA cost is 8,293"

Want to just do 8293?

Reply:
The "increased" gas cost I'm guessing is due to the recover of the signer.